File: Foundation/ValidTriangle.java
package Foundation;

import java.util.Scanner;

public class ValidTriangle {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int a = sc.nextInt();
		int b = sc.nextInt();
		int c = sc.nextInt();
		
        if ((checkValidity(a, b, c)) == 1)
            System.out.print("Valid");
        else
            System.out.print("Invalid");
	}
	
	public static int checkValidity(int a, int b, int c) {
		if(a+b <= c || a+c <=b || b+c <= a) {
			return 0;
		}
		return 1;
	}

}

File: Preparation_Python/2025/ValidTriangle.py
def validTriangle(a,b,c):
    if(a+b<=c or a+c <=b or b+c<=a):
        return False
    return True
a=int(input())
b=int(input())
c=int(input())
if validTriangle(a,b,c):
    print("Valid")
else:
    print("Not Valid")